projects/visual-regression/src/styles.scss

```diff
@@ -1 +1,27 @@
-@import '~@terminus/design-tokens/css/library-design-tokens.css';/* You can add global styles to this file, and also import other style files */
+@import '~@terminus/ui/helpers';
+@import '~@terminus/ui/terminus-ui.css';
+@import '~@terminus/design-tokens/css/library-design-tokens.css';
+
+html {
+  background-color: color(pure);
+  font-family: $type__family--base;
+  font-size: $type__size--base;
+  font-weight: $type__weight--base;
+}
+
+body {
+  margin: 0;
+}
+
+hr,
+.page {
+  margin: 1rem;
+}
+
+
+// stylelint-disable-next-line selector-type-no-unknown, scss/selector-nest-combinators
```

Comment:
Looks like VR app already has it's own stylelint config, so feel free to disable some rules like this that we may not care about in apps like this.